### PR TITLE
Add jakarta Feature

### PR DESCRIPTION
**Pull request by Fyro-Ing**
This is a proposition to include jakarta feature

https://github.com/swagger-api/swagger-codegen-generators/issues/1054

only add jakarta Feature

upgrade spring boot3, java17 are some other features that need to be on dedicated PR

### CODE REVIEW

**Comment by Fyro-Ing** [comment]
is there any chance to review this PR & merge for jakarta usage ?

**Comment by Fyro-Ing** [comment]
@HugoMario

**Comment by HugoMario** [comment]
Thanks a lot for this PR @Fyro-Ing !!

**Comment by VladPrudius** [comment]
Hello @HugoMario could you please clarify what is the status of this PR ?
We want to start migration to Spring boot 3, but we have blocker of swagger pluggin.

**Comment by HugoMario** [comment]
Hi @VladPrudius 
This is already merged. Let me know if you need any help.

**Comment by VladPrudius** [comment]
ah cool, as I understand it is enough just to add `jakarta` into additionalProperties section, correct ? I`ve already use latest version of swagger-codegen-cli:3.0.42 
@HugoMario 
image

**Comment by HugoMario** [comment]
Oh, but It wll be available in next release. Meantime you can use [this snapshot version](https://oss.sonatype.org/content/repositories/snapshots/io/swagger/codegen/v3/swagger-codegen-cli/3.0.43-SNAPSHOT/swagger-codegen-cli-3.0.43-20230424.195527-3.jar)

**Comment by VladPrudius** [comment]
Got it. Thank you for your time. 
One more thing, do you know when will be next release ? @HugoMario

File: src/main/java/io/swagger/codegen/v3/generators/handlebars/java/JavaHelper.java [hunk 1]
```diff
@@ -79,20 +79,26 @@ public CharSequence getModelImports(Map<String, Object> templateData, Options op
...
         }
         if (jackson && withXml) {
             builder.append("import com.fasterxml.jackson.dataformat.xml.annotation.*;\n");
         }
-        if (withXml) {
-            builder.append("import javax.xml.bind.annotation.*;\n");
+        if (withXml && jakarta) {
+            builder.append("import jakarta.xml.bind.annotation.*;\n");
+        } else if (withXml) {
+            builder.append("import jakarta.xml.bind.annotation.*;\n");
```
Comment (OleksandrShkurat) [inline src/main/java/io/swagger/codegen/v3/generators/handlebars/java/JavaHelper.java:92]: Looks like "import javax.xml.bind.annotation.*;\n" is expected here

Comment (Fyro-Ing) [reply src/main/java/io/swagger/codegen/v3/generators/handlebars/java/JavaHelper.java:92]: You're right
I'll try to push fix today

**Pull request merged**